### PR TITLE
Radius context config automated

### PR DESCRIPTION
**Pull request by mmoayyed**
Automates the configuration radius servers and clients, so that only declaring the authN handler is needed, and configuration is moved to the cas.properties. Adopters can still define additional servers and clients if they need to, just like before.

### CODE REVIEW

File: cas-server-documentation/installation/RADIUS-Authentication.md [hunk 1]
```diff
@@ -7,52 +7,47 @@ title: CAS - RADIUS Authentication
...
-      c:clientFactory-ref="radiusClientFactory" />
+The following settings may control the configuration RADIUS server and client:
+
+{% highlight properties %}
+# cas.radius.client.inetaddr=localhost
+# cas.radius.client.port.acct=
+# cas.radius.client.socket.timeout=60
+# cas.radius.client.port.authn=
+# cas.radius.client.sharedsecret=N0Sh@ar3d$ecReT
+# cas.radius.server.protocol=EAP_MSCHAPv2
```
Comment (spaetow) [inline cas-server-documentation/installation/RADIUS-Authentication.md]: I'm assuming that this includes other protocols (like EAP_TTLS_EAP_MSCHAPv2), yes?

Comment (mmoayyed) [inline cas-server-documentation/installation/RADIUS-Authentication.md]: Yes, everything that is supported by RadiusProtocol which are the following:

```java
    /** The chap. */
    CHAP("chap"),
    
    /** The EA p_ m d5. */
    EAP_MD5("eap-md5"),
    
    /** The EA p_ mscha pv2. */
    EAP_MSCHAPv2("eap-mschapv2"),
    
    /** The eap tls. */
    EAP_TLS("eap-tls"),
    
    /** The eap ttls pap. */
    EAP_TTLS_PAP("eap-ttls:innerProtocol=pap"),
    
    /** The EA p_ ttl s_ ea p_ m d5. */
    EAP_TTLS_EAP_MD5("eap-ttls:innerProtocol=eap-md5"),
    
    /** The EA p_ ttl s_ ea p_ mscha pv2. */
    EAP_TTLS_EAP_MSCHAPv2("eap-ttls:innerProtocol=eap-mschapv2"),
    
    /** The MSCHA pv1. */
    MSCHAPv1("mschapv1"),
    
    /** The MSCHA pv2. */
    MSCHAPv2("mschapv2"),
    
    /** The pap. */
    PAP("pap"),
    
    /** The peap. */
    PEAP("peap");

```

I will update the docs to make this known.

Comment (mmoayyed) [inline cas-server-documentation/installation/RADIUS-Authentication.md]: If you see one that is missing, let me know.

Comment (mmoayyed) [inline cas-server-documentation/installation/RADIUS-Authentication.md]: In fact, I don't see yours in the list: `EAP_TTLS_EAP_MSCHAPv2`. So will add :)

Comment (spaetow) [inline cas-server-documentation/installation/RADIUS-Authentication.md]: Actually it is. It's just before MSCHAPv1. However, it does not list EAP_TTLS_EAP_GTC, although I don't know whether JRadius supports EAP-GTC as an inner protocol...

Comment (mmoayyed) [inline cas-server-documentation/installation/RADIUS-Authentication.md]: OK, cool. I was too quick look :) In that case, I'd just leave it be, unless a specific request comes up.

**Pull request merged**